components/brave_wallet/browser/swap_service.cc

```diff
@@ -295,46 +302,10 @@ void SwapService::Bind(mojo::PendingReceiver<mojom::SwapService> receiver) {
   receivers_.Add(this, std::move(receiver));
 }
 
-void SwapService::SetBaseURLForTest(const GURL& base_url_for_test) {
-  base_url_for_test_ = base_url_for_test;
-}
-
-// static
-std::string SwapService::GetBaseSwapURL(const std::string& chain_id) {
-  if (!base_url_for_test_.is_empty()) {
-    return base_url_for_test_.spec();
-  }
-
-  if (chain_id == brave_wallet::mojom::kGoerliChainId) {
-    return brave_wallet::kGoerliSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kMainnetChainId) {
-    return brave_wallet::kSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kPolygonMainnetChainId) {
-    return brave_wallet::kPolygonSwapBaseAPIURL;
-  } else if (chain_id ==
-             brave_wallet::mojom::kBinanceSmartChainMainnetChainId) {
-    return brave_wallet::kBinanceSmartChainSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kAvalancheMainnetChainId) {
-    return brave_wallet::kAvalancheSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kFantomMainnetChainId) {
-    return brave_wallet::kFantomSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kCeloMainnetChainId) {
-    return brave_wallet::kCeloSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kOptimismMainnetChainId) {
-    return brave_wallet::kOptimismSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kArbitrumMainnetChainId) {
-    return brave_wallet::kArbitrumSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kSolanaMainnet) {
-    return brave_wallet::kSolanaSwapBaseAPIURL;
-  } else if (chain_id == brave_wallet::mojom::kBaseMainnetChainId) {
-    return brave_wallet::kBaseSwapBaseAPIURL;
-  }
-
-  return "";
-}
-
 // static
-GURL SwapService::GetZeroExQuoteURL(const mojom::SwapQuoteParams& params) {
+GURL SwapService::GetZeroExQuoteURL(
+    const mojom::SwapQuoteParams& params,
+    const std::optional<std::string> fee_param) {
```

Comment:
& for fee_param

Reply:
and everywhere in this file

Reply:
56d51daeb1738074c38dade4c96aec77d3863f8b